.github/workflows/version.yml

```diff
@@ -0,0 +1,23 @@
+name: version
+on:
+  release:
+    types:
+      - created
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Update SDK Version
+        run: |
+          # Extract the new tag name from the event payload
+          TAG_NAME=${{ github.event.release.tag_name }}
+
+          # Update the sdkVersion variable in deepgram.go file
+          sed -i "s/var sdkVersion string = .*/var sdkVersion string = \"$TAG_NAME\"/" deepgram/deepgram.go
```

Comment:
This should be updated for this new location:
https://github.com/deepgram/deepgram-go-sdk/blob/main/pkg/client/interfaces/utils.go#L18